docs/example-apps/django.md

```diff
@@ -0,0 +1,92 @@
+# Django
+
+Our example Django app can be found [here](https://github.com/convox-examples/django).
+
+The following is a step-by-step walkthrough of how the app was configured and why. The sample app is copied from the [Django 2.1 tutorial](https://docs.djangoproject.com/en/2.1/intro/tutorial01/), Dockerised, configured and a `convox.yml` added.  See the `README.md` in the repo for the changes made.
+
+### Running Locally
+
+Before we begin you will need to install the `convox` CLI and a development Rack:
+
+* [Command Line Interface](../installation/cli.md)
+* [Development Rack](../installation/development-rack)
+
+Once you are all setup you can switch to your local rack with ```convox switch [rack name]``` and start your local application with ```convox start``` (make sure you are in the root directory).
+
+Now that your app is up and running you will need to run the migrations with a [one-off command](/management/run):
+
+```bash
+convox run web python manage.py migrate
+```
+
+And create a super user for the Django Admin
+
+```bash
+convox run web python manage.py createsuperuser
+```
+
+You should now be able to access your application by going to [https://web.django.convox](https://web.django.convox). If you renamed anything you may need to modify your local URL. The format is https://[service name].[app name].convox
+
+### Custom Application Components
+
+#### Dockerfile
+
+Starting from the [python:3](https://hub.docker.com/_/python/) image, the [Dockerfile](https://github.com/convox-examples/django/blob/master/Dockerfile) executes the remaining build steps that your Django app needs. There are basically 2 steps in this process, and they are executed in a particular order to take advantage of Docker's build caching behavior.
+
+1. Requirements.txt is copied and `pip install` is run. This happens first because it is slow and something that's done infrequently. After running once, this step will be cached unless the cache is busted by later edits to `requirements.txt`.
+
+2. The application source is copied over. These files will change frequently, so this step of the build will very rarely be cached.
+
+#### convox.yml
+
+The [convox.yml](https://github.com/convox-examples/django/blob/master/convox.yml) file explains how to run the application. This file has two sections.
+
+1. Resources: These are network-attached dependencies of your application. In this case we have a single resource which is a postgres database. When [running locally](https://docs.convox.com/development/running-locally) Convox will automatically startup up a container running Postgres and will inject a ```DATABASE_URL``` environment variable into your application container that points to the Postgres database. When your application is [deployed](https://docs.convox.com/deployment/deploying-to-convox) to production Convox will startup an RDS postgres database for your application to use. 
```

Comment:
I would change this to when deployed instead of [running locally] and then delete everything after "...that points to the Postgres database" since we don't do the RDS thing on v3